ml-proto/spec/kernel.ml

```diff
@@ -119,7 +119,7 @@ and func' =
 type memory = memory' Source.phrase
 and memory' =
 {
-  initial : Memory.size;
+  min : Memory.size;
```

Comment:
Yes, fair point. The naming in BinaryFormat is more symmetric, so I chose that. We should resolve it either way, happy to change back if that's preferred.